ocaml/ocamltest/tsl_semantics.ml

```diff
@@ -92,6 +92,19 @@ type test_tree =
     string located list *
     (test_tree list)
 
+let tests_do_something (tests : Tests.t) =
+  List.exists Actions.does_something tests.test_actions
+
+let rec test_tree_does_something_on_all_branches tree =
+  match tree with
+  | Node (_, tests, _, children) ->
+    let from_children =
+      List.fold_left (fun acc child ->
```

Comment:
It looks like `List.exists`, but I
think I think I was expecting
a `List.for_all` (with the condition
over `tests` under `for_all`).

Reply:
This should be fixed as discussed